src/lib/registration/ui/addon_selection_base_dialog.rb

```diff
@@ -93,16 +93,51 @@ def details_widget
                 _("Select an extension or a module to show details here") + "</small>")))
       end
 
+      # @return [String] a Value for a RichText
+      def richtext_checkboxes(addons)
+        addons.map do |addon|
+          # checkbox label for an unavailable extension
+          # (%s is an extension name)
+          label = addon.available? ? addon.label : (_("%s (not available)") % addon.label)
+
+          rt_cb(id: addon_widget_id(addon),
+                label: label,
+                selected: addon_selected?(addon),
+                enabled: addon.available?,
+                indented: addon.depends_on)
+        end
+      end
+
+      IMG_ON = "/usr/share/YaST2/theme/current/wizard/checkbox-on.png"
```

Comment:
Um, I just realized that we need to handle a different UI theming in installation. So in `Stage.initial` we need to use https://github.com/yast/yast-theme/blob/master/SLE/wizard/inst_checkbox-on.png and https://github.com/yast/yast-theme/blob/master/SLE/wizard/inst_checkbox-off.png images.

#### Installation
![](https://github.com/yast/yast-theme/raw/master/SLE/wizard/inst_checkbox-on.png)
![](https://github.com/yast/yast-theme/raw/master/SLE/wizard/inst_checkbox-off.png)

#### Installed system
![](https://raw.githubusercontent.com/yast/yast-theme/master/SLE/wizard/checkbox-on.png)
![](https://raw.githubusercontent.com/yast/yast-theme/master/SLE/wizard/checkbox-off.png)